### DISCUSSION TITLE
Emmet support?

### DISCUSSION THREAD

**Discussion by keyboy250** [ajaxorg/ace#4473] [Ideas]
in Dreamweaver and brackets and code we can use Emmet to speed up wring code .
is there any support for this useful subset in Ace ?

https://docs.emmet.io/cheat-sheet/

**Comment by andrewnester** [comment]
we have an emmet support, see the demo https://ace.c9.io/demo/emmet.html

**Reply by ldijkman** [reply]
I only see a white page
How long do i have to wait? ;-)

**Reply by ldijkman** [reply]
why can i not find a single demo on the web with a working emmet code

**Reply by Syone** [reply]
Do you guys have the warning message: "script for emmet-core is not loaded"?
![image](https://github.com/ajaxorg/ace/assets/1608813/63296c2a-9989-4cb3-89e9-0df973042cb7)

**Reply by ldijkman** [reply]
Yes same
Never have seen a working ace with emmet

**Reply by Syone** [reply]
Ok it seems that it works even with this warning message. I haven't seen any documentation about emmet commands on ace editor, I had to take a look at the extension keymap to figure out how to use it: https://github.com/ajaxorg/ace/blob/master/src/ext/emmet.js#L294-L321

**Comment by keyboy250** [comment]
ace is integrated into godaddy cpanel, it is essential that emmet works in
the ace editor ...

On Mon, Sep 25, 2023 at 11:35 AM Dirk Luberth Dijkman <
***@***.***> wrote:

> I only see a white page
> How long do i have to wait? ;-)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ajaxorg/ace/discussions/4473#discussioncomment-7105015>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOCEZ3ALOSOZYIVAPN3C3TX4HFIPANCNFSM4Z5FAYCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,
Russ Dionne
